build.sbt

```diff
@@ -29,7 +30,7 @@ lazy val root = (project in file(".")).
     libraryDependencies += "org.apache.spark" %% "spark-core" % sparkVersion % "provided",
     libraryDependencies += "org.apache.spark" %% "spark-streaming" % sparkVersion % "provided",
     libraryDependencies += "org.apache.spark" %% "spark-sql" % sparkVersion % "provided",
-    libraryDependencies += "org.apache.spark" %% "spark-sql-kafka-0-10" % sparkVersion,
+    libraryDependencies += "org.apache.spark" %% "spark-sql-kafka-0-10" % sparkVersion exclude("net.jpountz.lz4", "lz4"),
```

Comment:
Can you add a comment about the exclusion?

Reply:
+1